src/types.rs

```diff
@@ -100,10 +162,11 @@ impl Response {
         // we should be handling based on headers but works for now
         self.response_type = "static_file".to_string();
         self.file_path = Some(file_path.to_string());
-        self.body = match read_file(file_path) {
+        let string = match read_file(file_path) {
```

Comment:
We can name this thing a bit better? Something like `string_response` or  just `response`?